apps/acrs/translations/src/en/buttons.json

```diff
@@ -2,8 +2,6 @@
     "start": "Start",
     "upload-file": "Upload",
     "save-and-exit": "Save and exit",
-    "delete": "Delete",
-    "remove": "Remove",
```

Comment:
there is a reference to this translation in `{{#t}}buttons.remove{{/t}}`
in `apps/acrs/views/upload-evidence.html`

Reply:
Added these back in. I had the remove button configuration in there originally but must have lost it from rebasing or pulling old code.